Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -618,6 +620,22 @@
                         sortable: false,
                         filterable: false
                     },
+                    {
+                        field: "LastChangedName", title: "Sidst redigeret: Bruger", width: 150,
+                        persistId: "lastchangedname", // DON'T YOU DARE RENAME!
+                        template: dataItem => dataItem && dataItem.LastChangedByUser ? `${dataItem.LastChangedByUser.Name} ${dataItem.LastChangedByUser.LastName}` : "",
```

Comment:
since you enforce the value in line 365, there is no need to check for null on LastChangedByUser